test/integration/submission/submission-schema-migration.spec.ts

```diff
@@ -23,10 +34,14 @@ chai.use(require('chai-http'));
 chai.use(require('deep-equal-in-any-order'));
 chai.should();
 
+// legacy field name
+const TUMOUR_STAGING_SYSTEM = 'tumour_staging_system';
 const schemaName = 'ARGO Clinical Submission';
 const startingSchemaVersion = '1.0';
 
 describe('schema migration api', () => {
+  buildDynamicStubSchema();
```

Comment:
I don't know this feels weird, do we need to build the migration stub schema every time the test is run. The dynamic schema building is cool since the stub schemas have to be updated with certain code changes... but it seems to me like this is a one time thing, no? Or is this something we want to do everytime.

Reply:
Hm, I'm not sure. On one hand, not building it every time saves on the unnecessary repeated computation. But the potential issue with not calling on this function every time is that if somebody makes changes to the `schema-builder.ts` (say to add another schema), then they also have to remember to actually run the script so the json file will be rewritten to. If they forget then things relying on the latest schema won't work, which they may not realize?

Maybe I could put a comment? Or do something else?

Reply:
But this process isn't so straight forward eh. It looks like you have to get the diffs from lectern and put them in stub-diffs.ts. So really you need to script/manually generate the stub-schemas anyways. There's a lot of places to forget to do something :stuck_out_tongue:.